CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Version 0.2.9
+
+- Fix the `clippy::needless_borrows_for_generic_args` lint in the README
+  example. (#38)
+- Swap `std::usize::MAX` for `usize::MAX`. (#39)
```

Comment:
The second one has no effect on the user, so I don't think mentioning it in changelog is necessarily necessary.